### PR TITLE
Added Flexible HB alerts doc

### PR DESCRIPTION
**Pull request by wawanawna**
Screenshot from 2020-05-29 13-28-52

### CODE REVIEW

File: docs/alerts/creating-heartbeat-alerts.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You can specify tags to filter and group by, then create Heartbeat Alerts for services matching your criteria. This way, even if a containerized service is moved from one host to another, it will not trigger false alerts.
+
+To make this possible Sematext Agent sends tags from the Sematext Common Schema that tag-based alerts use: 
+
+- os.host
+- container.id
+- container.image.name
+- container.image.tag
+- container.name
```
Comment (adnanrahic) [inline docs/alerts/creating-heartbeat-alerts.md:18]: Add backticks around these to add code formatting.

os.host -> `os.host`
etc...

File: docs/alerts/creating-heartbeat-alerts.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Using these tags you can do things like:
+- Filtering and grouping by container.image.name to get alerted when all containers with the specified image name are gone.
```
Comment (adnanrahic) [inline docs/alerts/creating-heartbeat-alerts.md:21]: container.image.name -> `container.image.name`

File: docs/alerts/creating-heartbeat-alerts.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- os.host
+- container.id
+- container.image.name
+- container.image.tag
+- container.name
+
+Using these tags you can do things like:
+- Filtering and grouping by container.image.name to get alerted when all containers with the specified image name are gone.  
+- Grouping by container.id to get alerted when that particular container dies. Yes, this could lead to a lot of alert noise, so use this sparingly and wisely.
```
Comment (adnanrahic) [inline docs/alerts/creating-heartbeat-alerts.md:22]: container.id -> `container.id`

File: docs/alerts/creating-heartbeat-alerts.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- Grouping by `os.host` and `container.image.name` to get alerted when a host stops running all containers with some image name. The container may have been moved to another host.
+
+This example will show how to include two specific images we’re interested in and grouping them by `os.host` and `container.image.name` to get alerted when a host stops running all containers with a particular image name.
+
+Open the Heartbeat Alert creation dialog. You’ll see that we group by the `os.host` tag by default. On the preview chart you may see a number of bars. These bars represent your hosts.
+
+![image alt text](../images/alerts/heartbeat_image_02.png)
+
+In the filter section, you see the ability to include and exclude tags. Let’s include two values for the container.image.name tag. Alerts will now only observe containers which were built from these 2 images.
```
Comment (adnanrahic) [inline docs/alerts/creating-heartbeat-alerts.md:31]: container.image.name -> `container.image.name`

**Comment by wawanawna** [comment]
> Some tiny formatting fixes.
> Also, please take a full screen screenshot of this docs page, and paste below, so we can see rendered formatting. You do this the same way you take the screenshot from Sematext Cloud. 1280x800 and press the `Capture Full Size Screenshot` button.

localhost_8000_alerts_creating-heartbeat-alerts_

File: docs/alerts/creating-heartbeat-alerts.md (hunk 1 above)
Comment (wawanawna) [reply docs/alerts/creating-heartbeat-alerts.md:18]: done

File: docs/alerts/creating-heartbeat-alerts.md (hunk 3 above)
Comment (wawanawna) [reply docs/alerts/creating-heartbeat-alerts.md:22]: done

File: docs/alerts/creating-heartbeat-alerts.md (hunk 4 above)
Comment (wawanawna) [reply docs/alerts/creating-heartbeat-alerts.md:31]: done

File: docs/alerts/creating-heartbeat-alerts.md (hunk 2 above)
Comment (wawanawna) [reply docs/alerts/creating-heartbeat-alerts.md:21]: done

File: docs/alerts/creating-heartbeat-alerts.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+To make this possible Sematext Agent sends tags from the Sematext Common Schema that tag-based alerts use: 
+
+- os.host
+- container.id
+- container.image.name
+- container.image.tag
+- container.name
+
+Using these tags you can do things like:
```
Comment (adnanrahic) [inline docs/alerts/creating-heartbeat-alerts.md:20]: Add an empty line between line 20 and 21. Formatting is broken.

**Pull request merged**